### PR TITLE
accounts via webauthn

### PR DESCRIPTION
**Pull request by tantommy**
- create accounts via hsm through webauthn api. these accounts are saved to idstore
- import hsm accounts with phrase or address
- refactor per changes from many-js
- cleanup/remove unused files

### CODE REVIEW

**Comment by tantommy** [comment]
@hansl , now ready for review

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "framer-motion": "^6.2.8",
     "idb-keyval": "^6.1.0",
     "localforage": "^1.10.0",
-    "many-js": "l-1-labs/many-js#75402779abd8f468de95cc910e9be9454a8f573a",
+    "many-js": "tantommy/many-js#32bfcff4e4e8ba3594ad640f7a3ea82ee932dca6",
```
Comment (hansl) [inline package.json:26]: Don't forget to update this

Comment (tantommy) [reply package.json:26]: atm, it is up to date with https://github.com/tantommy/many-js/commits/tt/webauthn PR, but i'll change it to the main branch from upstream whenever PR gets merged

Comment (tantommy) [reply package.json]: @hansl , has been updated and build output can be found here https://github.com/tantommy/albert/actions/runs/2386056231

**Review by hansl — APPROVED** [review]

**Pull request merged**